src/components/UserDashboard/ParticipantSets/index.tsx

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable react/prop-types */
+import * as React from 'react';
+import { FunctionComponent, useEffect } from 'react';
+import { Button, Table, Spin } from 'antd';
+import { DeleteFilled, EditFilled } from '@ant-design/icons';
+import { connect, ConnectedProps } from 'react-redux';
+import { RootState } from 'store/rootState';
+import { DispatchSaveSets, SaveSetState } from 'store/saveSetTypes';
+import {
+  selectErrorUserSaveSets,
+  selectIsLoadingSaveSets,
+  selectUserSaveSets,
+} from 'store/selectors/saveSetsSelectors';
+import { getUserSaveSets } from 'store/actionCreators/saveSets';
+
+import { AlignType } from 'rc-table/lib/interface';
+
+import './ParticipantSets.css';
+import { LoggedInUser } from 'store/userTypes';
+
+type OwnProps = {
+  user: LoggedInUser;
+};
+
+const mapState = (state: RootState): SaveSetState => ({
+  create: {
+    isLoading: false,
+    error: null,
+  },
+  userSets: {
+    isLoading: selectIsLoadingSaveSets(state),
+    sets: selectUserSaveSets(state),
+    error: selectErrorUserSaveSets(state),
+  },
+});
+
+const mapDispatch = (dispatch: DispatchSaveSets) => ({
+  userSaveSets: (userId: string) => dispatch(getUserSaveSets(userId)),
+});
+
+const connector = connect(mapState, mapDispatch);
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+type Props = PropsFromRedux & OwnProps;
+
+const align = 'right' as AlignType;
+
+const columns = [
+  {
+    title: 'Name',
+    dataIndex: 'name',
+    key: 'name',
+    // eslint-disable-next-line react/display-name
+    render: (name: string) => (
+      <div style={{ display: 'inline-flex', alignItems: 'center' }}>
+        <div className={'save-set-table-name'}>{name} </div>
+        <Button size={'small'} type={'text'}>
+          <EditFilled style={{ paddingLeft: 2.75 }} />
```

Comment:
css vs style

is it possible to have a fraction of an integer for a pixel? 2.75